entry: flow_with_init_exception:MyFlow
